packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -286,27 +291,19 @@ class Frame extends React.Component {
      */
     if (event && event.target
       && (event.target.hasAttribute('data-terra-form-select-toggle-button')
-      || event.target.hasAttribute('data-terra-form-select-toggle-button-icon'))) {
+      || event.target.hasAttribute('data-terra-form-select-toggle-button-icon')
+      || event.target.hasAttribute('data-terra-form-select-toggle')
+      || event.target.className.includes('arrow-icon'))) {
       this.setState({ isOpen: true, isPositioned: false });
 
-      // Allows time for state update to render select menu DOM before shifting focus to it
-      setTimeout(() => {
-        if (document.querySelector(this.selectMenu)) {
-          document.querySelector(this.selectMenu).focus();
-        }
-      }, 10);
+      FrameUtil.shiftFocusToMenu(this);
       return;
     }
 
     if (this.input) {
       this.input.focus();
     } else {
-      // Allows time for state update to render select menu DOM before shifting focus to it
-      setTimeout(() => {
-        if (document.querySelector(this.selectMenu)) {
-          document.querySelector(this.selectMenu).focus();
-        }
-      }, 10);
+      FrameUtil.shiftFocusToMenu(this.selectMenu);
```

Comment:
This probably isn't working correctly using the menu reference

Reply:
Forgot to revert to `this`, was testing passing selectMenu. Done b5171d2